concourse/pipelines/windows-image-build.jsonnet

```diff
@@ -456,8 +456,8 @@ local imgpublishjob = {
       passed: [
         job.passed,
       ],
-      // Builds are automatically pushed to testing. Triggering staging will automatically progress to prod and internal.
-      trigger: if job.env == 'staging' then false else true,
+      // Builds are automatically pushed to testing.
+      trigger: if job.env == 'testing' then true else false,
```

Comment:
@bkatyl This is what we do for Linux where only testing is automated (after build).

Reply:
Altered this to be:
build -> auto publish to testing
trigger publish to prod -> auto publish to internal prod